### DISCUSSION TITLE
Scheduled Update Summary message "You must provide at least ..."

### DISCUSSION THREAD

**Discussion by komir2004** [akeeba/panopticon#510] [Q&A (Community Support)]
Hello, I noticed a problem with the Scheduled Update Summary feature. 
When I try to use it, I get this error message: Akeeba\Panopticon\Library\Mailer\Mailer::Send failed: "You must provide at least one recipient email address.".
I don't know where to enter the email address. I have already added an email in the system configuration. 
Can you please help me fix this issue? Thank you.

**Comment by nikosdion** [comment]
This is already addressed. It happened when trying to send an email about a site, but there is no CC address defined for that site.
